src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,50 @@
+.calendar {
+  width: calendarWidth($calendar-size, $wrapper-gap, $wrapper-side-pading);
+  padding-top: 50px;
+
+  &__wrapper {
+    padding: 0 $wrapper-side-pading;
+    display: flex;
```

Comment:
After that couldbe used justify-content: center;
Also for centering your calendar